ase/Makefile

```diff
@@ -417,12 +424,12 @@ ifneq ($(SIMULATOR), VCS)
 endif
 # Check gate simulation libraries
 ifeq ($(GLS_SIM), 1)
-  ifndef QUARTUS_HOME
-	@echo "** ERROR **: env(QUARTUS_HOME) has not been set."
-	@echo "** ERROR **: GLS_SIM needs QUARTUS_HOME environment variable to be set."
+  ifeq ($(QUARTUS_HOME), )
```

Comment:
I don't think you need this complex check. Just confirm that QUARTUS_ROOTDIR is set. If the override is also set that won't be a problem, but QUARTUS_ROOTDIR is the sufficient requirement.

Then you can avoid setting QUARTUS_HOME above.

Reply:
Yes, you can use QUARTUS_ROOTDIR for those. I haven't seen any significant changes for the way we use them across many Quartus versions. It ought to be safe to ignore even the override.